compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/expression/FirGetClassCallChecker.kt

```diff
@@ -36,10 +35,25 @@ object FirGetClassCallChecker : FirBasicExpressionChecker() {
         //
         // Only the 2nd example is valid, and we want to check if token type QUEST doesn't exist at the same level as COLONCOLON.
```

Comment:
Maybe you can update the examples with type parameter examples (reified and not reified)?

Reply:
These are about unparsed `?`. Rather, I added some examples somewhere else when new diagnostics are added.